test/razorpay/test_payment.rb

```diff
@@ -23,6 +26,67 @@ def test_payments_should_be_available
       assert_equal 'card', payment.method, 'Payment method is accessible'
     end
 
+    def test_payments_fetch_downtime
+      stub_get( %r{payments/downtimes$}, 'downtimes_collection')
+      payments = Razorpay::Payment.fetchPaymentDowntime
+      assert_instance_of Razorpay::Collection, payments, 'Payment not an instance of Razorpay::Payment class'
+      assert !payments.items.empty?, 'downtimes should be more than one'
+    end
+    
+    def test_payments_fetch_downtime_by_id
+      stub_get( %r{payments/downtimes/#{@downtime_id}$}, 'fake_downtime')
+      payment = Razorpay::Payment.fetchPaymentDowntimeById(@downtime_id)
+      assert_instance_of Razorpay::Payment, payment, 'Payment not an instance of Razorpay::Payment class'
+      assert_equal @downtime_id , payment.id
+    end
+    
+    def test_payments_fetch_card_details
+      stub_get( %r{payments/#{@payment_id}/card$}, 'fake_card')
+      payment = Razorpay::Payment.fetchCardDetails(@payment_id)
+      assert_instance_of Razorpay::Card, payment, 'Payment not an instance of Razorpay::Payment class'
+      assert_equal @card_id , payment.id
+    end 
+
+    def test_payments_fetch_multiple_refund_for_payment
+      stub_get( %r{payments/#{@payment_id}/refunds$}, 'refund_collection')
+      payments = Razorpay::Payment.fetch(@payment_id).fetchMultipleRefund
+      assert_instance_of Razorpay::Collection, payments, 'Payment not an instance of Razorpay::Payment class'
+      assert !payments.items.empty?, 'downtimes should be more than one'
+    end
+
+    def test_payments_fetch_transfers
+      stub_get(%r{payments/#{@payment_id}/transfers$}, 'transfers_collection')
+      payments = Razorpay::Payment.fetch(@payment_id).fetchTransfer
+      assert_instance_of Razorpay::Collection, payments, 'Payment not an instance of Razorpay::Payment class'
+      assert !payments.items.empty?, 'payments should be more than one'
```

Comment:
This is a transfer collection, not payment

Reply:
Done